static/js/common/init.js

```diff
@@ -24,6 +26,45 @@ MEDUSA.common.init = function() {
         return offset;
     }
 
+    // function to change luminance of #000000 color - used in triggerhighlighting
+    function ColorLuminance(hex, lum) {
+        hex = String(hex).replace(/[^0-9a-f]/gi, '');
+        if (hex.length < 6) {
+            hex = hex[0]+hex[0]+hex[1]+hex[1]+hex[2]+hex[2];
+        }
+        lum = lum || 0;
+        var rgb = "#", c, i;
+        for (i = 0; i < 3; i++) {
+            c = parseInt(hex.substr(i*2,2), 16);
+            c = Math.round(Math.min(Math.max(0, c + (c * lum)), 255)).toString(16);
+            rgb += ("00"+c).substr(c.length);
+        }
+        return rgb;
+    }
+
+    // function to convert rgb(0,0,0) into #000000
+    function rgb2hex(rgb) {
+        rgb = rgb.match(/^rgb\((\d+),\s*(\d+),\s*(\d+)\)$/);
+        function hex(x) {
+            return ("0" + parseInt(x).toString(16)).slice(-2);
+        }
+        return "#" + hex(rgb[1]) + hex(rgb[2]) + hex(rgb[3]);
+    }
+
+    var revert_background_color; // used to revert back to original background-color after highlight
+    var allCells = $(".triggerhighlight");
```

Comment:
Why make a var if it's only used once? Just use the on with the jQuery selector.